src/index.html

```diff
@@ -1,12 +1,404 @@
+            <div class="card__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price-and-cost">
+            <p class="card__price">
+              Price:
+            </p>
+            <p class="card__cost">
+              $2,199
+            </p>
+          </div>
+
+          <a href="#"
+          class="card__button-buy"
+          data-qa="card-hover">
+            Buy
+          </a>
+
+        </div>
+
+        <div class="card">
+          <div class="card__image"></div>
+
+          <p class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars-and-reviews">
+            <div class="card__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price-and-cost">
+            <p class="card__price">
+              Price:
+            </p>
+            <p class="card__cost">
+              $2,199
+            </p>
+          </div>
+
+          <a href="#"
+          class="card__button-buy"
+          data-qa="hover">
+            Buy
+          </a>
+
+        </div>
+
+        <div class="card">
+          <div class="card__image"></div>
+
+          <p class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars-and-reviews">
+            <div class="card__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price-and-cost">
+            <p class="card__price">
+              Price:
+            </p>
+            <p class="card__cost">
+              $2,199
+            </p>
+          </div>
+
+          <a href="#"
+          class="card__button-buy"
+          data-qa="hover">
+            Buy
+          </a>
+
+        </div>
```

Comment:
no needs to add an empty line between a parent tag and child one
```suggestion
          </a>
        </div>
```